### DISCUSSION TITLE
help！

### DISCUSSION THREAD

**Discussion by cloudwalke** [windingwind/zotero-better-notes#1189] [Q&A] [answered]
每次当我打开这个插件的时候，笔记中的链接到原文的提示框便不再显示了，关了就好，一直没找到原因，把ZOTERO卸载重新安装也不好用。求助！这个真的非常影响使用
![000](https://github.com/user-attachments/assets/b2c7623b-e855-4e74-bec6-c6072af94eac)

**Comment by windingwind** [comment]
bug请提issue https://github.com/windingwind/zotero-better-notes/issues

**Comment by windingwind** [comment, marked as answer]
Update plugin

**Comment by cloudwalke** [comment]
Thank you so much！！！！！！！




------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2024年11月11日(星期一) 下午5:35
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [windingwind/zotero-better-notes] help！ (Discussion #1189)














 
Update plugin
